src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -136,7 +136,7 @@ public sealed override NamedTypeSymbol OriginalDefinition
             get { return _underlyingType; }
         }
 
-        internal sealed override NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved)
+        internal sealed override NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved, bool ignoreNonNullTypesAttribute)
         {
             return _unbound ? null : Map.SubstituteNamedType(OriginalDefinition.GetDeclaredBaseType(basesBeingResolved));
```

Comment:
>GetDeclaredBaseType(basesBeingResolved) [](start = 80, length = 39)

`GetDeclaredBaseType(basesBeingResolved, ignoreNonNullTypesAttribute)`

Reply:
Doesn't appear resolved?

---
In reply to: [201403500](https://github.com/dotnet/roslyn/pull/28320#discussion_r201403500) [](ancestors = 201403500)

Reply:
See earlier comment. I'm currently only passing `ignoreNonNullTypesAttribute` where it's known to avoid a cycle.